AddressInput/AddressInput.sol

```diff
@@ -0,0 +1,19 @@
+pragma ton-solidity >=0.35.0;
+
+interface IAddressInput {
+	function select(uint32 answerId, string prompt) external returns (address value);
```

Comment:
rename to get

Reply:
We've decided to use `select` because it's a complex UI component which allows to select address from different sources.